### PR TITLE
Important bug fix for User Signup. Mainly in views.py

### PR DESCRIPTION
**Pull request by HenryYao369**
1. + signup.html
	2. amend urls.py to apply changes
	3. add function in views.py to signup.

### CODE REVIEW

File: urls.py [hunk 1]
```diff
@@ -13,7 +13,8 @@
     #url(r'^about/(?P<pagename>[a-z\-]+)/$', 'kaeru.views.about_view'),
     url(r'^documentation/$' , 'kaeru.views.documentation_view'), 
     url(r'^login/$' , 'kaeru.views.login_view'), 
-    url(r'^logout/$' , 'kaeru.views.logout_view'), 
-    url(r'^people/$' , 'kaeru.views.people_view'),
-    url(r'^tutorial/$' , 'kaeru.views.tutorial_view'), 
+    url(r'^logout/$' , 'kaeru.views.logout_view'),
+    url(r'^signup/$' , 'kaeru.views.signup_view'),
+    # url(r'^people/$' , 'kaeru.views.people_view'),
```
Comment (fabianmuehlboeck) [inline urls.py]: Why is the people page commented out? Thus makes a test fail that otherwise passes.

Comment (HenryYao369) [inline urls.py]: Oh, it's when yesterday's debug, in order to go to admin site we commented 2 lines. In fact it should be tutorial's page that prevent us from going to admin site -- and maybe we should only comment this 1 line out?
So what I should only do is to revert the change to people page?

Comment (fabianmuehlboeck) [inline urls.py]: Yes

**Comment by Hapa29** [comment]
I actually got rid of the signup.html page and put the signup stuff on the landing page (index.html), so this is probably causing problems

**Comment by fabianmuehlboeck** [comment]
Yes, but Hengzhi re-added the signup page. That's fine for now as long as the functionality is fixed. The signup needs to be merged into index.html in the next step, though.

**Comment by HenryYao369** [comment]
Maybe in index.html there should be something else? Usually we don't put signup on landing page(for a website), but I don't know what your design/plan is...

**Comment by fabianmuehlboeck** [comment]
While your input is certainly welcome, this is definitely not the place and probably not the time for it. UI design and particularly the front page were discussed at length both in our meetings and in #6 .
